plugins/modules/provision_workflow_manager.py

```diff
@@ -933,56 +943,56 @@ class instance for further use.
                     self.log(self.msg, "ERROR")
                     self.status = "failed"
                     return self
+
+            if to_provisioning is True:
+                try:
+                    response = self.dnac_apply['exec'](
+                        family="sda",
+                        function="provision_wired_device",
+                        op_modifies=True,
+                        params=self.want["prov_params"],
+                    )
+                    self.log("Provisioning response collected from 'provision_wired_device' API is: {0}".format(response), "DEBUG")
+                except Exception as e:
+                    self.msg = "Error in provisioning due to {0}".format(str(e))
```

Comment:
Add the device details so that it will be easy to get the failure status for respective device.

Reply:
updated the code